resources/packs/aws/aws_ec2.go

```diff
@@ -1127,6 +1127,46 @@ func (p *mqlAwsEc2Volume) init(args *resources.Args) (*resources.Args, AwsEc2Vol
 	return nil, nil, errors.New("volume does not exist")
 }
 
+func (d *mqlAwsEc2Instance) init(args *resources.Args) (*resources.Args, AwsEc2Instance, error) {
+	if len(*args) > 2 {
```

Comment:
When I understand this correctly, I think `> 1` should be fine.

Reply:
that's in case someone wants to initialize with the instance id and region instead of the arn

Reply:
(these init functions are also used to initialize a resource from the context of another resource)